NineChronicles.Headless/NineChroniclesNodeService.cs

```diff
@@ -255,23 +257,20 @@ StandaloneContext context
             return service;
         }
 
-        internal static IBlockPolicy GetBlockPolicy(NetworkType networkType, IActionLoader actionLoader)
+        internal static IBlockPolicy GetBlockPolicy(Planet planet, IActionLoader actionLoader)
         {
             var source = new BlockPolicySource(actionLoader);
-            return networkType switch
+            return planet switch
             {
-                NetworkType.Main => source.GetPolicy(),
-                NetworkType.Internal => source.GetInternalPolicy(),
-                NetworkType.Permanent => source.GetPermanentPolicy(),
-                NetworkType.Test => source.GetTestPolicy(),
-                NetworkType.Default => source.GetDefaultPolicy(),
-                _ => throw new ArgumentOutOfRangeException(nameof(networkType), networkType, null),
+                Planet.Odin => source.GetPolicy(),
+                Planet.OdinInternal => source.GetInternalPolicy(),
+                _ => throw new ArgumentException(
+                    $"Can't retrieve policy for given planet ({planet})",
+                    nameof(planet)
```

Comment:
Is it okay for the heimdall network case?

Reply:
⌛

Reply:
@moreal I moved this dispatching to Lib9c side. PTAL https://github.com/planetarium/lib9c/pull/2326 🙏